test/engagement.e2e-spec.ts

```diff
@@ -491,6 +497,58 @@ describe('Engagement e2e', () => {
     ).toBeTruthy();
   });
 
+  it('should return empty methodologies array even if it is null', async () => {
+    const internshipProject = await createProject(app, {
+      type: ProjectType.Internship,
+    });
+
+    // Create InternshipEngagement without methodologies
+    const internshipEngagementInput: CreateInternshipEngagement = {
+      projectId: internshipProject.id,
+      internId: intern.id || (await createPerson(app)).id,
+    };
+
+    const result = await app.graphql.mutate(
+      gql`
+        mutation createInternshipEngagement(
+          $input: CreateInternshipEngagementInput!
+        ) {
+          createInternshipEngagement(input: $input) {
+            engagement {
+              id
+            }
+          }
+        }
+      `,
+      {
+        input: {
+          engagement: internshipEngagementInput,
+        },
+      }
+    );
+
+    const internshipEngagement: InternshipEngagement =
+      result.createInternshipEngagement.engagement;
```

Comment:
```suggestion
    // Create InternshipEngagement without methodologies
    const internshipEngagement = await createInternshipEngagement(app, {
      projectId: project.id,
      internId: intern.id,
      methodologies: [],
    });
```

Reply:
@CarsonF  Yes, that's a great idea. I changed